openquakeplatform/openquakeplatform/irv/static/irv/js/irv_viewer.js

```diff
@@ -90,10 +115,10 @@ function scaleTheData() {
     }
 
     // Populate the list with values
-    for (var key in indicatorsToBeScaled) {
-        for (var i = 0; i < layerAttributes.features.length; i++) {
-            var tempRegion = layerAttributes.features[i].properties[selectedRegion];
-            indicatorsToBeScaled[key][tempRegion] = layerAttributes.features[i].properties[key];
+    for (var ind in indicatorsToBeScaled) {
+        for (var i2 = 0; i2 < layerAttributes.features.length; i2++) {
+            var tempRegion = layerAttributes.features[i2].properties[selectedRegion];
+            indicatorsToBeScaled[ind][tempRegion] = layerAttributes.features[i2].properties[ind];
```

Comment:
`ind` and `i2` aren't more self-explaining than `key` and `i`, please use (at least) a couple of omogeneous names.